.github/workflows/docker-image.yml

```
@@ -0,0 +1,25 @@
+name: dockerHSSM CI
+
+on:
+  push:
+    branches: [ "main" ]
```

```suggestion
on:
  release:
    types:
      - created
```

We probably only want to update the `dockerfile` upon release